README.md

```diff
@@ -40,6 +40,11 @@ Calculator
 - Lift state up to Calculator
 - Pass state as props to Display
 
+### 4. Refactor with hooks
+
+- Convert class components to functional components
+- Implement state and memo hooks
+
```

Comment:
- [ ] In the README file, kindly fix the broken issues page link for a completely professional repo 😄 🙏